ui/src/templates/components/CommunityTemplateContents.tsx

```diff
@@ -22,6 +22,13 @@ interface StateProps {
 }
 
 class CommunityTemplateContentsUnconnected extends PureComponent<StateProps> {
+  private safelyAccessLength = collection => {
```

Comment:
this should be normalized to an empty array at the data layer (either right off the api call, before putting into redux, or when retrieving from redux, see @121watts's normalization work around resources) and not the view layer.

Reply:
@drdelambre i agree that what this code is doing isn't idiomatic or within the confines of our system, but that was a deliberate decision - we're still pretty working out what template data we're going to use or what shape it needs to be in or what parts of it is even useful to users right now. i don't think it makes sense to normalize until way later in this process. i think all that will do right now is slow down iteration and testing of this. this is still an mvp that we might still throw away.

Reply:
it would be super easy to just add this to the data layer.  also, easy to change if the shapes change / we need to rip it out.  also, if it _does_ stay it will be easier to extend and we wont have to back in and normalize after the fact.  if it's not normalized now, it probably wont get normalized.

Reply:
>if it's not normalized now, it probably wont get normalized.

based on what? if i needs to be normalized, i can normalize it before we officially launch this feature.

this probably doesn't even belong in redux state. this is transitional data and state - it's something that exists as a confirmation before we install the resources the template describes, which are what the users actually care about.